src/comms.cpp

```diff
@@ -1045,19 +1114,20 @@ void sec1_process_message(uint8_t key, uint8_t value)
     }
 }
 
+_millis_t wpDiscTime;
+
```

Comment:
The only place `wpDiscTime` is used is inside `transmitSec1()` function and then also only when SEC1_DISCONNECT_WP is defined.  Move this declaration to be a static inside that function and wrap in a #ifdef.